src/hooks/useSwapCallArguments.tsx

```diff
@@ -167,15 +169,16 @@ export function useSwapCallArguments(
       ]
     }
   }, [
-    trade,
-    recipient,
-    library,
     account,
```

Comment:
You should add sorting to eslint in a separate PR so that your PRs don't add additional churn.